idl/generateClient.js

```diff
@@ -8,7 +9,16 @@ const PROGRAM_NAME = "phoenix_v1";
 
 async function main() {
   generateTypeScriptSDK().then(() => {
-    conoole.log("done");
+    console.log("Running prettier on generated files...");
+    spawn("prettier", ["--write", sdkDir], { stdio: "inherit" })
+      .on("error", (err) => {
+        console.error(
+          "Failed to lint client files. Try installing prettier (`npm install --save-dev --save-exact prettier`)"
```

Comment:
Good call, if you have one, can you quickly paste to this PR. Not opinionated here

Reply:
just leave the comment for now and we can add one in if a future diff comes in that looks ugly. As long as the default includes semicolons and double-quotes since that's what all of our editors currently do